### PR TITLE
Testing KTX2

### PR DESCRIPTION
**Pull request by emackey**
Trying to get this working...

### CODE REVIEW

File: pages/babylonView.js [hunk 1]
```diff
@@ -79,6 +79,26 @@
...
+                jsDecoderModule: previewPath + 'babylon.ktx2Decoder.js',
+                wasmUASTCToASTC: previewPath + 'ktx2Transcoders/uastc_astc.wasm',
+                wasmUASTCToBC7: previewPath + 'ktx2Transcoders/uastc_bc7.wasm',
+                wasmUASTCToRGBA_UNORM: previewPath + 'ktx2Transcoders/uastc_rgba32_unorm.wasm',
+                wasmUASTCToRGBA_SRGB: previewPath + 'ktx2Transcoders/uastc_rgba32_srgb.wasm',
+                jsMSCTranscoder: previewPath + 'ktx2Transcoders/msc_basis_transcoder.js',
+                wasmMSCTranscoder: previewPath + 'ktx2Transcoders/msc_basis_transcoder.wasm'
+            };
+
+            //BABYLON.ZSTDDecoder.WasmModuleURL = previewPath + 'zstddec.wasm';
```
Comment (emackey) [inline pages/babylonView.js:100]: @bghgary It looks like ZSTDDecoder.WasmModuleURL is actually used someplace down in a worker thread, not in the main thread as with these other URLs.  Is there any way to configure it from here?

Comment (bghgary) [reply pages/babylonView.js:100]: @Popov72 Can you help?

Comment (Popov72) [reply pages/babylonView.js:100]: This PR will let you configure this url too: https://github.com/BabylonJS/Babylon.js/pull/9780

Comment (emackey) [reply pages/babylonView.js:100]: Awesome, thanks @Popov72 and @bghgary.

**Pull request closed without merging**